Source/Phone/PhoneSettings.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2016 Cisco Systems Inc
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+import Foundation
+
+
+/// Global settings for the default phone audio and video modes. These settings
+/// are not persisted.
+public class PhoneSettings {
+
+    /// Default camera facing mode, used as the default when dialing or answering a call.
+    public static var defaultFacingMode = Call.FacingMode.User
```

Comment:
It's more that it was always global, and unlike most things we didn't make it non-global. It's kind of a weird case. I can't say I'm thrilled to have it be global, but I'm not thrilled about it existing at all. It's non-persistent settings! That should be SOMETHING else. I'm not sure what. This is one of those things that wasn't getting in our way, so we didn't put much time into restructuring it.